tests/features/bootstrap/FeatureContext.php

```diff
@@ -1109,7 +1110,41 @@ public function assertPageNotCached() {
    *
    * @Given I select/check the :row_text row
    */
-  public function assertSelectRow($text) {
+  public function assertSelectRow(string $text): void {
```

Comment:
If we are touching this then we should take the opportunity to update the method name. Methods that are named `assertSomething()` are intended for checking a certain condition. They should throw an exception if the condition is not met.

I know this is also checking something, but there is a difference between checking a condition and checking a checkbox :)